pyuploadtool/releases_hosting_provider/github_releases.py

```diff
@@ -55,7 +55,7 @@ def create_release(self, metadata: ReleaseMetadata, artifacts):
             self.logger.warning("push to default branch, creating continuous release")
 
             # not using "latest", as this value is reserved by GitHub
-            metadata.tag = "continuous"
+            metadata.tag = os.getenv("ROLLING_RELEASE_TAG", "continuous")
```

Comment:
I guess the name should have a `GITHUB_` prefix, because you might use multiple release hosting providers in parallel.